### PR TITLE
[Fix-2248][admin][metadata] Modification only supports clickhouse driver version 0.2.6

### PR DESCRIPTION
**Pull request by LiuHao0606**
1.Upgrade the clickhouse driver to version 0.6.2
2.When modifying the connection data source, only the clickhouse driver version 0.2.6 is supported
3.The driver class for ClickHouse has been changed from ru.yandex.clickhouse.ClickHouseDriver to com.clickhouse.jdbc.ClickHouseDriver

### CODE REVIEW

File: dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java [hunk 1]
```diff
@@ -46,33 +49,51 @@ public class ClickHouseTest {
...
+        return Driver.buildUnconnected("ck","Clickhouse", map);
+//        return Driver.build(config);
     }
 
+
     @Ignore
     @Test
     public void connectTest() {
         String test = getDriver().test();
+        System.out.println(test);
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java:82]: use LOGGER.info()

File: dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java [hunk 2]
```diff
@@ -46,33 +49,51 @@ public class ClickHouseTest {
...
     @Test
     public void schemaTest() {
-        List<Schema> schemasAndTables = getDriver().getSchemasAndTables();
+        Driver driver = getDriver();
+        String test = driver.test();
+        Driver connect = driver.connect();
+
+
+        List<Schema> schemasAndTables = driver.getSchemasAndTables();
+        System.out.println(JsonUtils.toJsonString(schemasAndTables));
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java:96]: use LOGGER.info()

File: dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java [hunk 3]
```diff
@@ -81,7 +102,10 @@ public void schemaTest() {
     @Test
     public void columnTest() {
         Driver driver = getDriver();
-        List<Column> columns = driver.listColumns("xxx", "xxx");
+        String test = driver.test();
+        Driver connect = driver.connect();
+        List<Column> columns = driver.listColumns("xx", "xx");
+        System.out.println(JsonUtils.toJsonString(columns));
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java:108]: use LOGGER.info()

File: dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java [hunk 4]
```diff
@@ -90,7 +114,10 @@ public void columnTest() {
     @Test
     public void queryTest() {
         Driver driver = getDriver();
-        JdbcSelectResult query = driver.query("select * from xxx", 10);
+        String test = driver.test();
+        Driver connect = driver.connect();
+        JdbcSelectResult query = driver.query("select count(1) from xx.xx", 10);
+        System.out.println(query.getRowData());
```
Comment (Zzm0809) [inline dinky-metadata/dinky-metadata-clickhouse/src/test/java/org/dinky/metadata/ClickHouseTest.java:120]: use LOGGER.info()

**Comment by LiuHao0606** [comment]
close

**Pull request closed without merging**